picky-asn1-x509/src/subject_public_key_info.rs

```diff
@@ -40,6 +40,12 @@ impl SubjectPublicKeyInfo {
             ),
         }
     }
+    pub fn new_ec_key<P: Into<BitStringAsn1>>(ec_point: P) -> Self {
```

Comment:
Nit: spacing
```suggestion

    pub fn new_ec_key<P: Into<BitStringAsn1>>(ec_point: P) -> Self {
```

Reply:
It is done in this [commit](https://github.com/Devolutions/picky-rs/pull/110/commits/1441c85d96c538a323eb74c12476152acc55cf62)